WD-Firmware/rtos/rtosal/api_inc/rtosal_util_api.h

```diff
@@ -75,8 +75,9 @@ void rtosalStart(rtosalApplicationInit_t fptrInit);
 void rtosalParamErrorNotifyFuncSet(rtosalParamErrorNotification_t fptrRtosalParamErrorNotification);
 
 /**
-* This function is invoked by the system timer interrupt
+* This function implements ending of scheduler activity
```

Comment:
to be moved to rtosal_task.h

Reply:
For the error-notification handling I created new file - rtosal_error.c